examples/otel-logs-splunk-raw-hec/README.md

```diff
@@ -0,0 +1,31 @@
+# Splunk Raw HEC example
+
+This example showcases how the collector can expose a HTTP endpoint to receive data over the HEC endpoint as a raw payload.
+
+To learn more about HEC raw capabilities, head on to the [HEC documentation](https://docs.splunk.com/Documentation/Splunk/8.2.6/Data/FormateventsforHTTPEventCollector#Raw_event_parsing).
+
+The example runs as a Docker Compose deployment.
+
+Splunk is configured to receive data from the OpenTelemetry Collector using the HTTP Event collector. To learn more about HEC, visit [our guide](https://dev.splunk.com/enterprise/docs/dataapps/httpeventcollector/).
+
+To deploy the example, check out this git repository, open a terminal and in this directory type:
+```bash
+$> docker-compose up
+```
+
+Splunk will become available on port 18000. You can login on [http://localhost:18000](http://localhost:18000) with `admin` and `changeme`.
```

Comment:
Suggested change: "Splunk becomes available on port 18000. Log in to http://localhost:18000 using admin and changeme."